content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,91 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning">
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
 
 ## Compatibility
+Datadog .NET application performance monitoring supports a robust set of features to provide critical information about .NET applications & services: 
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+- Monitor [runtime metrics][1] to help troubleshoot bugs and detect resource inefficiencies in your application.
```

Comment:
My preference is to remove these new bullets and have the compatibility page be as minimal as possible, that solely answer the question of "Will this run on my machines?"

Reply:
Interesting, I do agree on the point of making it as clear as possible. However, having the capabilities section include the Runtime Metics & logs injection, which are  unique differentiators to our competitors does also serve a purpose. My goal with this is to help enable global customers, sales teams in different regions, and support engineers with all the features the .NET APM library has to offer.

Reply:
I guess this depends on whether we think this page should be focused on sales/new prospects, or whether it's devs looking to implement it. My assumption is the latter, and so I'm inclined to agree with Zach, especially under the "compatibility" title

Reply:
Thanks team, I agree. I will add a new section for Capabilities above the Compatibility section. I will draft up a PR now - again, thank you!

Reply:
Update: the changes have been added to the following [PR](https://github.com/DataDog/documentation/pull/11200/commits/3cbbb246f859f9a4102decfd6272a0a510cd777a).  I moved the Capabilities section to the main docs and left the Compatibility minimal. 

Marking this as resolved.